.scalafmt.conf

```diff
@@ -1,6 +1,6 @@
 version = 3.7.2
 runner.dialect = scala213
-maxColumn = 150
+maxColumn = 180
```

Comment:
This deviates from the standard in Confluence. Is there a reason for this?

Reply:
No particular reason.  I do think we should avoid trying to explicitly enforce *exactly* the same formatting rules across all repos as its not really necessary to achieve the benefits and its also impossible as these things will drift.
I think the one in confluence should be regarded as the minimum base formats